.github/workflows/ci.yml

```diff
@@ -1,7 +1,6 @@
 name: Tests
 
-on:
-  push
+on: push
```

Comment:
Oh I think we need `on: [push, pull_request]` since push on works on branches *on this repo* and yours is a fork obviously.

Reply:
Eureka - that fixed! Thanks Scott. CI appears happy.